src/data/data.js

```diff
@@ -0,0 +1,182 @@
+export default [
```

Comment:
Gatsby uses a datasource system, you don't have to map csv to json yourself: https://whatjackhasmade.co.uk/using-a-csv-file-as-a-data-source-in-gatsby

Reply:
Thx. :)

Reply:
If i need to transform the CSV, how do i do that ? :(

Under the hood, the package `gatsby-transformer-csv` uses `node-csvtojson`.

https://github.com/Keyang/node-csvtojson#hook--transform

Reply:
I think the point here is to easily mount the csv and use graphql then to read it; you can after map the result to whatever you want 🤔

Reply:
Doc is here: https://www.gatsbyjs.com/docs/content-and-data/

Reply:
Yes. But i might be better to have the data formatted from the start.

Reply:
With your solution, the non-formatted data will be packaged in the distribution and will be fixed at run time by a `.map()` function.

Reply:
In other words, how to transform the CSV at built time ?

Reply:
I don't know :) I suggest to use Gatsby way to consume data source but if it's not appropriate we may do it your way.

Reply:
Ok, for now, let's do it the Gatsby way :)